repos/system_upgrade/el7toel8/actors/vsftpdconfigcheck/libraries/library.py

```diff
@@ -26,5 +26,9 @@ def check_config_supported(tcpwrap_facts, vsftpd_facts):
             reporting.ExternalLink(
                 title='Replacing TCP Wrappers in RHEL 8',
                 url='https://access.redhat.com/solutions/3906701'
-            )
-        ])
+            ),
+            reporting.RelatedResource('file', '/etc/hosts.allow'),
+            reporting.RelatedResource('file', '/etc/hosts.deny'),
+            reporting.RelatedResource('package', 'tcp_wrappers'),
+            reporting.RelatedResource('package', 'vsftpd'),
+        ] + [reporting.RelatedResource('file', bc) for bc in bad_configs])
```

Comment:
Are you sure we need to refer to bad_configs here?

Reply:
Hmm, it seems that `bad_configs` is basically a set of `/etc/hosts.{deny,allow}`, in that case I'd rather drop 30+31 and actually keep `bad_configs` so that we report only what is affected, what do you think?

Reply:
Yeah, probably that's better